nautobot/dcim/filters/__init__.py

```diff
@@ -1628,6 +1629,11 @@ class PowerPanelFilterSet(LocatableModelFilterSetMixin, NautobotFilterSet):
         to_field_name="name",
         label="Rack group (name or ID)",
     )
+    panel_type = django_filters.MultipleChoiceFilter(
+        choices=PowerPanelTypeChoices,
+        null_value=None,
+    )
+    breaker_position_count = django_filters.NumberFilter()
```

Comment:
This should be a nautobot `MultiValueNumberFilter` instead so that users can filter by multiple counts simultaneously.

It **may** be the case that you can remove this explicit declaration and get the correct automatic filter by including it in `fields` below.